e2e/rbd.go

```diff
@@ -791,6 +791,112 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("test RBD volume ecryption with user secrets based SecretsMetadataKMS", func() {
+				err := deleteResource(rbdExamplePath + "storageclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete storageclass with error %v", err)
+				}
+				scOpts := map[string]string{
+					"encrypted":       "true",
+					"encryptionKMSID": "user-ns-secrets-metadata-test",
+				}
+				err = createRBDStorageClass(f.ClientSet, f, defaultSCName, nil, scOpts, deletePolicy)
+				if err != nil {
+					e2elog.Failf("failed to create storageclass with error %v", err)
+				}
+
+				// user provided namespace where secret will be created
+				namespace := "rook-ceph"
```

Comment:
use `rookNamespace` variable and why we are creating secret in a rook-ceph namespace?

Reply:
>use rookNamespace variable 

:+1: ok

>and why we are creating secret in a rook-ceph namespace? 

This e2e test case tests fetching of secret from user specified namespace `user-ns-secrets-metadata-test` . 
Used `default` earlier, but during deployment its replaced with e2e testing namespace.